### PR TITLE
handle the situation when extension has been installed

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: microsoft/testsuites/vm_extensions/AzureMonitorAgentLinux.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def verify_azuremonitoragent_linux(self, log: Logger, node: Node) -> None:
 
         try:
             # Delete VM Extension if already present
-            extension.delete("AzureMonitorLinuxAgent")
+            extension.delete("Microsoft.Azure.Monitor.AzureMonitorLinuxAgent")
+            is_extension_present = True
         except HttpResponseError as identifier:
             if any(s in str(identifier) for s in ["was not found"]):
```
Comment (squirrelsc) [inline microsoft/testsuites/vm_extensions/AzureMonitorAgentLinux.py:61]: Please output log in the `else` part as well, even I'm thinking the test case should fail, if it's not "was not found".

File: microsoft/testsuites/vm_extensions/azsecpack.py [hunk 2]
```diff
@@ -161,25 +166,36 @@ def _install_monitor_agent_extension(self, node: Node) -> None:
...
-            auto_upgrade_minor_version=True,
-            enable_automatic_upgrade=True,
-            settings=settings,
-        )
-        assert_that(result["provisioning_state"]).described_as(
-            "Expected the extension to succeed"
-        ).is_equal_to("Succeeded")
+        try:
+            result = extension.create_or_update(
+                name="AzureMonitorLinuxAgent",
```
Comment (squirrelsc) [inline microsoft/testsuites/vm_extensions/azsecpack.py:171]: Should the name be changed like "Microsoft.Azure.Monitor.AzureMonitorLinuxAgent"?

Comment (LiliDeng) [reply microsoft/testsuites/vm_extensions/azsecpack.py:171]: confirmed with Siddhartha, he said either name is okay.

File: microsoft/testsuites/vm_extensions/AzureMonitorAgentLinux.py (hunk 1 above)
Comment (LiliDeng) [reply microsoft/testsuites/vm_extensions/AzureMonitorAgentLinux.py:61]: updated

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**